wagtail/admin/templates/wagtailadmin/base.html

```diff
@@ -26,6 +26,20 @@
                                     {% icon name=level_tag classname="messages-icon" %}
                                 {% endif %}
                                 {{ message|safe }}
+                                {% if level_tag == "error" %}
+                                    {# Show a button to focus the first field/response error if available. #}
+                                    <button
+                                        type="button"
+                                        class="button button-small button-secondary w-invisible"
+                                        data-controller="w-count w-focus"
+                                        data-action="click->w-focus#focus"
+                                        data-w-count-active-class="!w-visible"
+                                        data-w-count-find-value=".error-message,.help-critical"
```

Comment:
Double check this for potential hidden message issues. 

See https://github.com/wagtail/wagtail/pull/13371